contrib/exporters/secadvisor/mod/resolve_runc.go

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2019 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy ofthe License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specificlanguage governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package mod
+
+import (
+	"github.com/spf13/viper"
+
+	"github.com/skydive-project/skydive/api/client"
+	"github.com/skydive-project/skydive/contrib/exporters/core"
+	g "github.com/skydive-project/skydive/gremlin"
+)
+
+// NewResolveRunc creates a new name resolver
+func NewResolveRunc(cfg *viper.Viper) (interface{}, error) {
+	gremlinClient := client.NewGremlinQueryHelper(core.CfgAuthOpts(cfg))
+
+	return &resolveRunc{
+		gremlinClient: gremlinClient,
+	}, nil
+}
+
+type resolveRunc struct {
+	gremlinClient *client.GremlinQueryHelper
+}
+
+// IPToName resolve ip to name
+func (r *resolveRunc) IPToName(ipString, nodeTID string) (string, error) {
+	node, err := r.gremlinClient.GetNode(g.G.V().Has("Runc.Hosts.IP", ipString).ShortestPathTo(g.Metadata("TID", nodeTID)))
+	if err != nil {
+		return "", err
+	}
+
+	name := node.Metadata["Runc"].(map[string]interface{})["Hosts"].(map[string]interface{})["Hostname"].(string)
+	return "0_0_" + name + "_0", nil
```

Comment:
why these prefix, sufix ?

Reply:
this is the current format used for security advisor :-(